lib/jsmidgen.js

```diff
@@ -408,6 +408,15 @@ var Midi = {};
 		return bytes;
 	};
 
+	File.prototype.setTicks = function(num) {
+		// Do nothing in case of invalid input
+		if (!num || typeof num !== 'number') {
+			return;
+		}
+
```

Comment:
Also, there should probably be a range check to ensure that the number of ticks is > 0 and < `1<<15`, which is the maximum allowed by the MIDI format. If the high bit is 1 (i.e. the number of ticks is `1<<16`) then it will be interpreted differently.